nixos/modules/services/scheduling/fcron.nix

```
@@ -73,7 +73,7 @@ in
       systab = mkOption {
         type = types.lines;
         default = "";
-        description = ''The "system" crontab contents.'';
+        description = "The "system" crontab contents.";
```

```suggestion
        description = "The \"system\" crontab contents.";
```
That should fix the evaluation error.